gammapy/background/tests/test_ring.py

```diff
@@ -34,6 +35,32 @@ def test_run(self):
         assert_allclose(result['off'].data[0, 0], 248.0)
 
 
+@requires_dependency('scipy')
+class TestAdaptiveRingBackgroundEstimator:
+    def setup(self):
+        opts = dict(r_in=0.22 * u.deg, r_out_max=0.8 * u.deg, width=0.1 * u.deg)
+        self.ring = AdaptiveRingBackgroundEstimator(**opts)
+        self.images = SkyImageList()
+
+        self.images['counts'] = SkyImage.empty(nxpix=101, nypix=101, fill=1)
+        self.images['exposure_on'] = SkyImage.empty(nxpix=101, nypix=101, fill=1E10)
+        exclusion = SkyImage.empty(nxpix=101, nypix=101, fill=1)
+        exclusion.data[40:60, 40:60] = 0
+        self.images['exclusion'] = exclusion
+
+
+    def test_run(self):
+        result = self.ring.run(self.images)
+        assert_allclose(result['background'].data[50, 50], 1)
+        assert_allclose(result['alpha'].data[50, 50], 0.002638522427440632)
```

Comment:
Is there a way to debug if that alpha value is correct?
What matters is to be able to figure out which scale index was used for the pixel and how many pixels the ring kernel for that scale has.
Possible to access for experts / debugging after running the algorithm?